src/worker-thread/canvas/WebGLRenderingContextPolyfill.ts

```diff
@@ -1968,73 +2173,83 @@ export class WebGLRenderingContextPolyfill extends GLConstants implements WebGL2
     transfer(this.canvas.ownerDocument as Document, [TransferrableMutationType.OBJECT_MUTATION, fnName, this, args]);
   }
 
-  private _bindBuffer(target: GLenum, buffer: vGLBuffer | null): void {
-    switch (target) {
-      case this.ARRAY_BUFFER: {
-        this._buffers.arrayBuffer = buffer;
-        break;
-      }
-      case this.ELEMENT_ARRAY_BUFFER: {
-        this._buffers.elementArrayBuffer = buffer;
-        break;
-      }
-      case this.COPY_READ_BUFFER: {
-        this._buffers.copyReadBuffer = buffer;
-        break;
-      }
-      case this.COPY_WRITE_BUFFER: {
-        this._buffers.copyWriteBuffer = buffer;
-        break;
-      }
-      case this.TRANSFORM_FEEDBACK_BUFFER: {
-        this._buffers.transformFeedbackBuffer = buffer;
-        break;
-      }
-      case this.UNIFORM_BUFFER: {
-        this._buffers.uniformBuffer = buffer;
-        break;
-      }
-      case this.PIXEL_PACK_BUFFER: {
-        this._buffers.pixelPackBuffer = buffer;
-        break;
+  _webglError(errorCode: GLenum, errorName: string, operation: string, message: string): void {
+    this._error = errorCode;
+    console.warn(`WebGL: ${errorName}(${errorCode}): ${operation}: ${message}`);
+  }
+
+  _validateBinding(candidate: TransferrableGLObject | null, validator: Function, operation: string, expectedType: string, index: number): boolean {
```

Comment:
explicit access modifier is missed. the default one in TS is public.

Reply:
done